docs/2.0/api/size.md

```diff
@@ -34,6 +34,7 @@ Sets how the image is fitted to its target dimensions.
 - `contain`: Default. Resizes the image to fit within the width and height boundaries without cropping, distorting or altering the aspect ratio.
 - `max`: Resizes the image to fit within the width and height boundaries without cropping, distorting or altering the aspect ratio, and will also not increase the size of the image if it is smaller than the output size. 
 - `fill`: Resizes the image to fit within the width and height boundaries without cropping or distorting the image, and the remaining space is filled with the background color. The resulting image will match the constraining dimensions.
+- `fill-max`: Resizes the image to fit within the width and height boundaries with cropping but without distorting the image. The resulting image will match the constraining dimensions.
```

Comment:
The code checks for `fillmax` without the `-`. So the docs and code need to be harmonized. 

Personally I would prefer using `fill-max` since other multi word values for other options are also dash separated, for e.g. `top-left`.

Reply:
I have updated all other references to fill-max

Reply:
> .. but without distorting the image

This seems a bit inaccurate. With `fill-max` smaller images will be stretched isn't it (while still maintaining aspect ratio)? So technically that will distort/blur the image.

Reply:
Yeah it's inaccurate. I should change it to without cropping but with distorting. And add that the remaining width or height will be filled with a color